prov/gni/test/rdm_atomic.c

```diff
@@ -481,7 +484,7 @@ void do_min(int len)
 
 	cr_assert_eq(ret, 1, "fi_cq_read returned %d %d", ret,
 			dump_cq_error(send_cq[0], target, 0));
-	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0);
+	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0, ep[1]);
```

Comment:
We don't test this code path in many places, so I think at least some of them should change to get a remote event.

The RMA tests are set up for it (in fact, some of them check already), so I'd add it to those and skip the atomic tests for now.  You can add an issue to add remote CQ event checking for the atomic tests.